core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -472,7 +472,7 @@ shadow_stack_access_ok:
 	flush_cpu_semaphores
 	wait_secondary
 
-#ifdef CFG_PL310_LOCKED
+#if defined(CFG_PL310_LOCKED) && !defined(CFG_PL310_SIP_PROTOCOL)
```

Comment:
Can discard this change and simply insure `CFG_PL310_LOCKED` is not enabled for the platform.

Reply:
OK